### DISCUSSION TITLE
Which trustworthy client-app?

### DISCUSSION THREAD

**Discussion by B0rner** [dani-garcia/vaultwarden#4050] [Q&A]
I'm new to this topic. So sorry for that noob-question.
Currently I only use the vaultwarden webinterface to store login credentials for use in lokal environment,

But now, I like to access the the credentials by using the smartphone. Because the vaultwarden installation is only avalaible in local network, i need an vaultwarden client app, that is able to sync or cache the treasure in order to use it on the way without having access to the vaultwarden installation. 
Are there any ideas for an trustworthy app? Ideally one, that is encryptable with an yubikey.

**Comment by BlackDex** [comment]
Just use the Official Bitwarden Apps.
https://bitwarden.com/download/

**Reply by B0rner** [reply]
Is the sourcecode available to see what is happening under the hood?

**Reply by stefan0xC** [reply]
Yes. See https://github.com/bitwarden/clients/ and https://github.com/bitwarden/mobile/

**Reply by zatricky** [reply]
On the other hand:
https://github.com/bitwarden/clients/issues/11611

**Reply by mucookul** [reply]
> On the other hand:
> [bitwarden/clients#11611](https://github.com/bitwarden/clients/issues/11611)

It seems to have been fixed?
